test/performance/odp_l2fwd.c

```diff
@@ -1507,6 +1512,8 @@ static void usage(char *progname)
 	       "  -x, --vec_size <num>    Vector size (default %i).\n"
 	       "  -z, --vec_tmo_ns <ns>   Vector timeout in ns (default %llu ns).\n"
 	       "  -P, --promisc_mode      Enable promiscuous mode.\n"
+	       "  -l, --packet_len <len>  Maximum length of packets supported(default length 1536).\n"
+	       "  -L, --seg_len <len>     Packet pool segment length(default length 1536).\n"
```

Comment:
The defaults here should be handled the same way as in the other options.

Reply:
ACK